### DISCUSSION TITLE
Crashing on instancialize ChromiumWebBrowser

### DISCUSSION THREAD

**Discussion by marcussacana** [cefsharp/CefSharp#4731] [General]
The crash happens in the native library `libcef`, It is an access violation, and in IDA PRO I was able to caught where it happens
![image](https://github.com/cefsharp/CefSharp/assets/10576957/1aa45a3f-0834-4d87-9187-d4984c758377)
the string above should help localize the code

cefsharp version 121.3.130 x64

**Comment by amaitland** [comment]
Check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

See also https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#unmanaged-crashes-when-the-process-dies

> The crash happens in the native library `libcef`

CEF has it's own support forum at https://www.magpcss.org/ceforum/index.php

Youi can ask for help there.
